client/readme.md

```diff
@@ -0,0 +1,37 @@
+# Elm Frontend Readme
+
+The Lunar Rocks frontend is an Elm 0.19 application.
+
+==================================================
+
+## Setup
+
+Install npm packages from the `client` directory.
+```
+npm install
+```
+
+Install [Elm](https://guide.elm-lang.org/install.html) and [elm-Live](https://github.com/wking-io/elm-live), and [uglify-js](https://www.npmjs.com/package/uglify-js).
```

Comment:
Do these need to be installed manually by the user, or will they be installed via the `npm install` command? This is unclear.

If they need to be installed manually, could these all be installed with a single command?

e.g. `npm install -g elm elm-live uglify-js`?